flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/catalog/PostgresCatalogTestBase.java

```diff
@@ -37,9 +40,12 @@
 /** Test base for {@link PostgresCatalog}. */
 public class PostgresCatalogTestBase {
 
+    public static final Logger LOG = LoggerFactory.getLogger(PostgresCatalogTestBase.class);
+
     @Rule public ExpectedException exception = ExpectedException.none();
 
-    @ClassRule public static SingleInstancePostgresRule pg = EmbeddedPostgresRules.singleInstance();
+    protected static final DockerImageName POSTGRES_96_IMAGE =
+            DockerImageName.parse("postgres:9.6.12");
```

Comment:
We should define the used version like the others in https://github.com/apache/flink/blob/master/flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/util/DockerImageVersions.java